src/helpers/isPortTaken.js

```diff
@@ -0,0 +1,23 @@
+import net from 'net'
+
+export const isPortTaken = async (port, opts) => {
```

Comment:
Would it be better to use https://www.npmjs.com/package/tcp-port-used ? While there's value in not depending on useless dependencies, a non-trivial like this one reduces the amount of code to maintain going forward.